### PR TITLE
Simplify the travis build matrix

### PR DESCRIPTION
**Pull request by cbeer**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,14 +2,10 @@ language: ruby
 sudo: false
 script: rake
 rvm:
-  - 2.2.0
-  - 2.1.5
-  - 2.0.0
-  - 1.9.3
-  - jruby-1.7.9-d19 # fails after 1.7.10 for some reason
-#  - jruby-19mode # JRuby in 1.9 mode
+  - 2.2.3
```
Comment (ndushay) [inline .travis.yml]: Is 2.2.3 also going to be on harvestdor-xxx boxes?  We need to cover the spotlight boxes and the harvestdor-xxx boxes, but that should be it, I think?  

@peetucket  Does Revs index from harvestdor-xxx boxes?  Does it use harvestdor-indexer?

Comment (peetucket) [inline .travis.yml]: No, revs now uses revs-indexing-service boxes, with the new StanfordSync stack (discovery_indexer, base_indexer, etc.)  The old revs code is still on the harvestdor-xxx boxes and probably still works, but it is not updated, used or maintained anymore.

**Comment by ndushay** [comment]
looks like harvestdor-dev is at 2.2.0 -- maybe leave that in for now so you can proceed?

**Comment by cbeer** [comment]
Nah, it's easy enough to upgrade it. In any case, there's no reason to think that 2.2.0 is any different (for our purposes).

**Comment by ndushay** [comment]
It worries me that some tests on one of these PRs failed for ruby 2.2.0

**Comment by cbeer** [comment]
The failure in #19 is just a (new) rubocop violation that is fixed by this PR.

**Comment by ndushay** [comment]
:+1: 

do you want to make a ticket to update harvestdor-dev, or are you just gonna do it?

**Comment by cbeer** [comment]
Already done. SAR-3005.

**Pull request merged**